dts/bindings/sensor/maxim,max30208.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2021, Silvano Cortesi
```

Comment:
This binding needs to justify its use of defaults; see https://docs.zephyrproject.org/latest/guides/dts/bindings.html#general-rules for details.